### DISCUSSION TITLE
How to use the lightbox inline?

### DISCUSSION THREAD

**Discussion by stonesthatwhisper** [dromru/react-photoswipe-gallery#1070] [Q&A]
I could not find an example of using this inline. Photoswipe has an inline example but it looks quite different from anything that works with react.

**Comment by mattbarnicle** [comment]
what do you mean by inline?

**Comment by stonesthatwhisper** [comment]
I meant having the photoswipe fill the parent div instead of the full
window.

**Comment by akhmadullin** [comment]
@stonesthatwhisper, can you provide a link to photoswipe's "inline example"?

I can't figure out what you are talking about.

**Comment by stonesthatwhisper** [comment]
Please look at an example from another package here:
https://yet-another-react-lightbox.com/plugins/inline

**Comment by mattbarnicle** [comment]
How about using an iFrame?

**Comment by akhmadullin** [comment]
@stonesthatwhisper, as far as I know, [Photoswipe](https://photoswipe.com/) does not provide a way to open a gallery inside some container.

It has [appendToEl option](https://photoswipe.com/options/#appendtoel), but this option only mounts gallery to some element, but gallery still will be opened for the whole screen.
